poshc2/server/C2Server.py

```diff
@@ -401,6 +377,19 @@ def newdb(db):
     create_self_signed_cert(PoshProjectDirectory)
     newPayload.WriteQuickstart(directory + '/quickstart.txt')
 
+    # adding default hosted payloads
+    QuickCommandURI = select_item("QuickCommand", "C2Server")
+    insert_hosted_file("%ss/86/portal" % QuickCommandURI, "%sSharp_v4_x86_Shellcode.bin" % (PayloadsDirectory), "text/html", "Yes", "Yes")
```

Comment:
Can maybe make the Yes/No stuff a boolean in the DB

Reply:
Definitely a later refactor